src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/LibraryController.kt

```diff
@@ -12,12 +14,32 @@ import java.util.*
 @RestController
 @RequestMapping("/libraries")
 class LibraryController(private val libraryService: LibraryService) {
-  @GetMapping("/")
-  fun getLibraries(): ResponseEntity<List<Library>> {
+  @GetMapping("/all")
```

Comment:
is there a reason to do all vs just the endpoint?

Reply:
Nope, not sure why I added that - will take it out!